docs/reuse/ui-elements.md

```diff
@@ -4,5 +4,19 @@
 | ![icon-favorites.png](/img/content-sharing/icon-favorites.png) | Favorites | Content you’ve marked as favorite. |
 | ![icon-personal.png](/img/content-sharing/icon-personal.png) | Personal | Content you manage. |
 | ![icon-library.png](/img/content-sharing/icon-library.png) | Library | List of all accessible content, including your creations and the content shared with you. |
+| <img src="/img/content-sharing/icon-more-actions-kebab-three-dot.png" alt="more-actions-kebab" width="20%" /> | More Actions <!-- Kebab menu? Three-dot kebab icon? More Actions? -->  | Click to reveal a menu with additional options. |
+| <img src="/img/content-sharing/nav-menu.png" alt="nav-menu.png" /> | Navigation Menu  | Main Sumo Logic menu containing your collectors, sources, library, help, and more. |
+
+<!-- Use this after Kanso GA
```

Comment:
@kimsauce, I changed the tag from `<!--Kanso` to `<!-- Use this after Kanso GA` so that I wouldn't inadvertently remove the `<!--Kanso` tag at Kanso GA with DOCS-207.

Reply:
thank you, @jpipkin1 !